docs/190-paas-operations/upgrade/100-cluedin-upgrade-guide.md

```diff
@@ -403,34 +404,132 @@ After about **10 minutes**, the Helm command in PowerShell should complete and d
+
+During this upgrade, the UI will be disabled. Please notify all users in advance so they are aware of the downtime. 
+
+A standard CluedIn upgrade typically results in 20–30 minutes of downtime. If the upgrade includes data migrations or additional updates or large volumes of data, the outage may take longer. 
+
+**Verify the Current CluedIn Helm Chart Version** 
+Run the following command to list all Helm releases in the cluedin namespace: 
 
 ```powershell
 helm list -a -n cluedin 
 ```
 
-The output should display the new installed chart version. For example: 
+The output will display the currently installed chart version. For example: 
 
 ```powershell
-cluedin-platform-2.5.2 
+cluedin-platform-2.5.1 
 ```
 
-The following sections outline common operational tasks, along with typical failure scenarios and their mitigations.
+This indicates that the current Helm chart version in use is **2.5.1**. 
+
+```powershell
+# ${CustomValues} refers to the values file you have amended with the above changes. Please type the full path here. 
+
+helm upgrade cluedin-platform -n cluedin cluedin/cluedin-platform \
+     --version {CluedInChartVersion} \ 
+     --values ${CustomValues} \ 
+     --set application.system.upgradeMode=true \
+     --set application.system.runDatabaseJobsOnUpgrade=true \
+     --set application.system.runNugetFullRestore=true \
+     --wait \
+     --timeout 10m0s
+```
+
+**Parameter details:** 
+  - {CluedInChartVersion} – The target chart version specified in the documentation. 
+  - ${CustomValues} – The full path to the user values file you created in the previous step.
+  - upgradeMode=true - Ensures no data is ingested or processed 
+  - runDatabaseJobsOnUpgrade=true – Ensures the database is refreshed during the upgrade. 
+  - runNugetFullRestore=true – Ensures all packages are fully restored. 
+  - --timeout 10m0s – A best practice is to allow a 10-minute timeout. If the upgrade fails due to a timeout, please follow the documented mitigation steps. 
+
+After a few minutes, it should successfully complete.
+
+**Validation**
+  - Wait until deployment finishes. Make sure that all pods are healthy and all jobs are completed.
+  - Don’t proceed further if any of the pods have health issues. 
+
+**Run Data Upgrade**
+The data upgrade requires a trigger which is applied by running the following command:
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: api.cluedin.com/v1
+kind: DataUpgrade
+metadata:
+  annotations:
+    meta.helm.sh/release-name: cluedin-platform
+    meta.helm.sh/release-namespace: cluedin
+  labels:
+    app: cluedin
+    app.kubernetes.io/instance: cluedin-platform
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: application
+    helm.sh/chart: application-2.5.1
+    release: cluedin-platform
+  name: data-upgrade-450
+  namespace: cluedin
+spec:
+  toVersion: Version450
+EOF
+```
+
+**Validation**
+Inspect the CluedIn pod logs for the server ...
```

Comment:
@bkanda @Dizzudin46 @Saifulmrz  can you remember what exactly we should see in the logs?